README.md

```diff
@@ -59,6 +59,25 @@ Read [How the topic taxonomy works](https://docs.publishing.service.gov.uk/manua
 Content Tagger has a "Projects" function for tagging content in bulk
 to the Topic Taxonomy. It also supports tagging individual pages.
 
+### Links
+
+Content Tagger allows you to define your own concepts via 'links', which
```

Comment:
I'm not sure what you mean by "concepts" here.

I think this README is getting a bit crowded now. Perhaps Tagging to a facet-group, and the description of links-based facets could move to it's own page.

Reply:
[From yesterday](https://github.com/alphagov/finder-frontend/pull/1203#discussion_r296653184):

> links is an overloaded term and also applies to organisations, taxons etc. We're only talking about content tagged to links-based-facets

I'm trying to clarify that content tagger is not prescriptive and is used for lots of things. Not quite sure how to phrase it though.

Agreed that this section weighs the README down a bit, think it's worth keeping here but slimming down if possible 🤔

Reply:
This section should be called "Facets" then to line-up with the view in the app. See screenshot: https://docs.publishing.service.gov.uk/manual/tag-content-with-facet-values.html

"Content tagger can create facet groups. It also allows you to tag content to a facet group."

Reply:
Done 👍